cmd/clusterctl/cmd/create_cluster.go

```diff
@@ -118,6 +126,7 @@ func init() {
 
 	// Optional flags
 	createClusterCmd.Flags().StringVarP(&co.AddonComponents, "addon-components", "a", "", "A yaml file containing cluster addons to apply to the internal cluster")
+	createClusterCmd.Flags().StringVarP(&co.BootstrapOnlyComponents, "bootstrap-only-components", "", "", "A yaml file containing components to apply only on the bootstrap cluster but not the provisioned cluster")
```

Comment:
Should this also tell the user that it is applied before the providerComponents?

Reply:
In addition to the above, I believe the namespace object is part of the `providerComponents`, what if some of the bootstrap-only components needs to live in that namespace?

Reply:
It should be fine to apply the Namespace twice, once in the providerComponents and once in the bootstrapOnlyComponents.

Reply:
That's a good idea.  Changing the text.